src/pymor/models/transfer_function.py

```diff
@@ -554,52 +554,62 @@ class FactorizedTransferFunction(TransferFunction):
     """
 
     def __init__(self, dim_input, dim_output, K, B, C, D, dK=None, dB=None, dC=None, dD=None,
-                 parameters={}, sampling_time=0, name=None):
+                 parameters={'s': 1}, sampling_time=0, name=None):
```

Comment:
`s` should not be a parameter of transfer functions (or `LTIModel`), it should only be used internally.